gtk/src/toga_gtk/widgets/activityindicator.py

```diff
@@ -16,7 +16,8 @@ def stop(self):
         self.native.stop()
 
     def rehint(self):
-        # print("REHINT", self, self.native.get_preferred_width(), self.native.get_preferred_height())
+        # print("REHINT", self, self.native.get_preferred_width(),
+        # self.native.get_preferred_height())
```

Comment:
Even if commented out, code should still be at least flake8-compliant, otherwise it becomes difficult to read.